bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/dto/PersistenceItemInfoBean.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence.dto;
+
+/**
+ * This is a java bean that is used to serialize information about items in a persistence service to JSON.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class PersistenceItemInfoBean {
```

Comment:
Call it DTO, not Bean. So did you come to the conclusion that you definitely need it?